plugins/services/src/js/columns/QuotaOverviewLimitColumn.tsx

```diff
@@ -0,0 +1,78 @@
+import { ServiceGroup } from "#PLUGINS/services/src/js/types/ServiceGroup";
+import { Icon, TextCell, Tooltip } from "@dcos/ui-kit";
+import {
+  greyLight,
+  blue,
+  iconSizeXs
+} from "@dcos/ui-kit/dist/packages/design-tokens/build/js/designTokens";
+import { SystemIcons } from "@dcos/ui-kit/dist/packages/icons/dist/system-icons-enum";
+import * as React from "react";
+import { Trans } from "@lingui/macro";
+
+export function limitRenderer(group: ServiceGroup) {
+  const limit = group.quota ? group.quota.limitStatus : "N/A";
+  const roles =
+    group.quota && group.quota.serviceRoles
+      ? group.quota.serviceRoles.count
+      : 0;
+  const groupRoles =
+    group.quota && group.quota.serviceRoles
+      ? group.quota.serviceRoles.groupRoleCount
+      : 0;
+  const limitNumber = roles - groupRoles;
+
+  let icon = null;
+  if (limit === "Enforced") {
```

Comment:
if we changed to a `switch-case` here, we would be notified by typescript whenever the `ServiceGroupQuotaLimit`-type is extended (or a case is removed). The compiler would force us to handle (or remove) that case here, which would be great regarding maintainability! :)